lib/octokit/client/organizations.rb

```diff
@@ -595,6 +595,89 @@ def remove_organization_membership(org, options = {})
         user = options.delete(:user)
         user && boolean_from_response(:delete, "orgs/#{org}/memberships/#{user}", options)
       end
+
+      # Initiates the generation of a migration archive.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @option options [Array<String>] :repositories Repositories for the organization.
```

Comment:
The list of repositories is required by the API. Would it make sense to move the list of repositories to a normal argument (`def start_migration(org, repositories, options = {})`)?